jenkins/common.sh

```diff
@@ -9,4 +9,4 @@ echo $PWD
 echo TEST_HOME=${TEST_HOME}
 export PATH=${PATH}:${TEST_HOME}/apps/arakoon/bin
 
-make
+make clean build
```

Comment:
This will probably break parallel test runs

Reply:
iirc each build has its own copy of the workspace (which is where the build artifacts are produced), so this shouldn't be a problem?